streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableKTableJoinMerger.java

```diff
@@ -28,22 +28,52 @@
     private final String queryableName;
     private boolean sendOldValues = false;
 
-    public KTableKTableJoinMerger(final KTableImpl<K, ?, V> parent1,
-                                  final KTableImpl<K, ?, V> parent2,
-                                  final String queryableName) {
+    KTableKTableJoinMerger(final KTableImpl<K, ?, V> parent1,
+                           final KTableImpl<K, ?, V> parent2,
+                           final String queryableName) {
         this.parent1 = parent1;
         this.parent2 = parent2;
         this.queryableName = queryableName;
     }
 
     @Override
     public Processor<K, Change<V>> get() {
-        return new KTableKTableJoinMergeProcessor<>();
+        return new KTableKTableJoinMergeProcessor();
     }
 
     @Override
     public KTableValueGetterSupplier<K, V> view() {
```

Comment:
Would it be possible to add a test for this class or do you feel it is covered elsewhere?

Reply:
I think it is covered in `KTableKTableJoinTest#testJoin`.